acceptance/testdata/detector/Dockerfile

```diff
@@ -3,6 +3,8 @@ FROM ubuntu:bionic
 ARG cnb_uid=1234
 ARG cnb_gid=1000
 
+ENV CNB_PLATFORM_API=0.6
```

Comment:
I wasn't sure how the acceptance test are supposed to be targeting the latest platform API. I did this for now, but would love to know if there is a more automatic way so we don't have to bump these dockerfiles that I might've missed.

Reply:
I don't think we have an established pattern for this. Maybe we could make the api version an ARG - and then when we `h.DockerBuild` the image, we could pass `h.WithArgs("--build-arg", fmt.Sprintf("platformAPI=%s", api.Platform.Latest()))`. Individual tests could override the platform api version by passing `--env` when we run the image.